raw_data/preprocessed/africa/tanzania_english.txt

```diff
@@ -0,0 +1,769 @@
+
+Duty of local government authority in relation to hazardous waste 
+(134) [1] Each local government authority shall, with respect to its area of jurisdiction, ensure that: [a] standards prescribed for the hazardous waste management are in place and operational at all the time; [b] premises producing hazardous wastes are adequately ventilated and are in compliance with prescribed standards; [c] waste effluents are treated or are so modified as to comply with prescribed standards before final disposal; and [d] hazardous liquid wastes are treated to conform with prescribed environmental standards at factory or on site before their discharge into public sewers or municipal oxidation ponds or in an open land or into receiving water bodies. [2] Any standards and guidelines set by the local government for the purposes of this Act, shall conform to standards set under this Act. 
+
+Movement of hazardous waste 
+(135) [1] The Minister shall ensure that, any movement of hazardous waste within and through Tanzania shall be conducted in a manner that prevents or minimizes adverse effects to human health and the environment and shall conform to movement procedures as may be prescribed in the Regulations. [2] Any generator of hazardous waste shall take measures to minimize the generation of such waste. [3] Any generator of hazardous waste shall be responsible for its disposal and shall be liable for any damage to human health, living beings and the environment. 
+
+Environment Impact Assessment for hazardous waste 
+(136) [1] Subject to the provisions of subsection [2], disposal of any hazardous wastes shall be done in an environmentally sound manner. [2] Environmental Impact Assessment shall be carried out before hazardous waste is disposed of into soil, land, air or body of water. 
+
+Final disposal of health care wastes 
+(137) [1] The Minister shall, in collaboration with the Minister responsible for health, ensure that health care wastes are sorted and stored in prescribed coded containers and transported in refuse trucks designed and registered for that purpose. [2] The Minister shall in collaboration with the Minister responsible for health, prescribe the best possible method for final disposal of various types of health care wastes. [3] In this section, "health care wastes" includes but is not limited to the infectious wastes, pathological wastes, sharps, pharmaceutical, genetoxic, radioactive wastes, coagulated blood wastes and. 
```

Comment:
Please replace `coagulated blood wastes and` with `coagulated blood wastes and drugs`.